docs/_static/mxnet-theme/index.html

```diff
@@ -17,27 +17,26 @@
     </div>
   </div>
 </div>
-
 <div class="section-tout">
   <div class="container">
     <div class="row">
       <div class="col-lg-4 col-sm-12">
-        <h3>Apache MXNet 1.0 Released</h3>
-        <p>We're excited to announce the release of MXNet 1.0! Check out the release notes for latest updates.</p>
-        <a href="https://github.com/apache/incubator-mxnet/releases/tag/1.0.0">Learn More</a>
+        <h3>Apache MXNet 1.0.1 Released</h3>
+        <p>We're excited to announce the release of MXNet 1.0.1! Check out the release notes for latest updates.</p>
+        <a href="https://github.com/apache/incubator-mxnet/releases/tag/1.0.1">Learn More</a>
       </div>
       <div class="col-lg-4 col-sm-12">
-        <h3>Introducing - Gluon</h3>
-        <p>We’re happy to introduce a new elegant, easy to use, imperative interface for MXNet.
+        <h3>MXNet Model Server</h3>
+        <p>Apache MXNet Model Server (MMS) is a flexible and easy to use tool for serving deep learning models exported from MXNet or the Open Neural Network Exchange (ONNX).
```

Comment:
This is not an apache package. @hen would that be a problem?

Reply:
Please change to:
Model Server for Apache MXNet

Reply:
And get rid of the (MMS) since it doesn't match up here! Thanks!